e2e/_suites/metricbeat/metricbeat_test.go

```diff
@@ -378,14 +378,14 @@ func (mts *MetricbeatTestSuite) runMetricbeatService() error {
 			return err
 		}
 
-		err = docker.LoadImage(imagePath)
+		err = deploy.LoadImage(imagePath)
```

Comment:
I think these should stay in the docker namespace, the deployer is an abstraction on top of docker/k8s and I think this ties deployer to much to docker. I think a better approach is to have a deploy.LoadImage that supports both docker and k8s way of importing images into a registry